addons/survey/static/src/js/survey.js

```diff
@@ -230,4 +230,15 @@ if(!the_form.length) {
         }
     });
 
+    $('a').each(function(key, value) {
```

Comment:
The jquery selector could be better: `$('.breadcrumb-item a')`. So you get only the breadcrumb links at once, without having to filter afterwards.

Moreover, I'd move that part before `$('.js_surveyform').ajaxFor`, just for logical organization.